frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js

```diff
@@ -85,10 +85,8 @@ class DataCube {
     this.isSegmentation = isSegmentation;
 
     this.MAX_UNSAMPLED_ZOOM_STEP = resolutionsLength - 1;
+    this.ZOOM_STEP_COUNT = resolutionsLength;
```

Comment:
`MAX_ZOOM_STEP` and `MAX_UNSAMPLED_ZOOM_STEP` now are the same afaics. Do we even need the `MAX_UNSAMPLED_ZOOM_STEP` any longer?

Reply:
Good catch! I had removed all usages, but forgot to remove the definition :see_no_evil: